src/popup/App.vue

```diff
@@ -132,10 +157,16 @@ export default defineComponent({
       await watchUntilTruthy(() => isRestored.value);
 
       setNotificationSettings();
+      initNotifications();
```

Comment:
Maybe instead of always turning on the notifications polling the composable can decide on this. There are screens that don't need the notifications (like for example the web iframe popups).

The composable can track how many components are using them at a moment and when the number goes to 0 the polling can be turned off. This way the App should not import the notifications composable.